src/components/Popup.js

```diff
@@ -3,18 +3,18 @@ export default class Popup {
         this._popup = document.querySelector(popupSelector);
     }
 
-    open() {
+    open(){
         this._popup.classList.add("modal_opened");
-        document.addEventListener("keydown", this._handleEscClose);
+        document.addEventListener("keydown", (e)=>{
+           if(e.key === "Escape"){
+            this.close();
+           }

```

Comment:
Please, return the right code back. You had `document.addEventListener("keydown", this._handleEscClose);` here